src/test/java/rv/atqc/gttesting/archex/PlacesPage.java

```diff
@@ -0,0 +1,132 @@
+package rv.atqc.gttesting.archex;
+
+import java.util.ArrayList;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+public class PlacesPage extends CategoriesPage{

```

Comment:
In theory is possible because places, tracks, and events it is dynamic parts of category, but for simplify code, i think it will be better if "PlacePage extends Abstract Page", and if you want use your cod you need add "generic" in class CategoryPage